Source/lighting.cpp

```diff
@@ -322,22 +322,22 @@ void MakeLightTable()
 {
 	// Generate 16 gradually darker translation tables for doing lighting
 	uint8_t shade = 0;
-	constexpr uint8_t black = 0;
-	constexpr uint8_t white = 255;
+	constexpr uint8_t BLACK = 0;
+	constexpr uint8_t WHITE = 255;

```

Comment:
Shouty case is usually reserved for preprocessor stuff

Reply:
capitalize the first letter then? VS2019 is angry because they were constexpr and lowercase

Reply:
Most of the code style rules are defined in Source/.clang-tidy, constants use `CamelCase`
https://github.com/diasurgical/devilutionX/blob/0ac30d101dc6ab0adf33772dc1595d7529a2556e/Source/.clang-tidy#L77-L81